gm4_crossbow_cartridges/data/gm4/advancements/crossbow_cartridges_string.json

```diff
@@ -0,0 +1,33 @@
+{
+    "display": {
+        "icon": {
+            "item": "string",
+            "nbt": "{display:{Name:\"\\\"crossbow_cartridges logo\\\"\"},CustomModelData:1}"
```

Comment:
Custom model data ID is fine, I just added it to the document. But we no longer need the display name.